tests/unit/dataactvalidator/test_a34_appropriations.py

```diff
@@ -0,0 +1,33 @@
+from tests.unit.dataactcore.factories.domain import SF133Factory
+from tests.unit.dataactcore.factories.staging import AppropriationFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+
+_FILE = 'a34_appropriations'
+_TAS = 'a34_appropriations_tas'
+
+
+def test_column_headers(database):
+    expected_subset = {'row_number', 'budget_authority_unobligat_fyb', 'sf_133_amount'}
+    actual = set(query_columns(_FILE, database))
+    assert (actual & expected_subset) == expected_subset
+
```

Comment:
Am loving these unit tests for the rules...so easy to read! Do you think we need a test case for the scenario of a new tas/new fiscal year where we expect the rule to pass where there's no corresponding SF-133 record? Is that taking things too far?